File: Easy/Sorting - Bubble Sort/countSwaps.py
def countSwaps(a):
    count = 0
    for _ in range(len(a)):

        for j in range(len(a) - 1):
            
            if a[j] > a[j+1]:
                temp = a[j]
                a[j] = a[j+1]
                a[j+1] = temp
                count += 1
    
    print('Array is sorted in {} swaps.'.format(count))
    print('First Element: {}'.format(a[0]))
    print('Last Element: {}'.format(a[-1]))

File: Easy/Sorting - Bubble Sort/countSwaps.js
function countSwaps(a) {
    let count = 0

    for (let _ of a) {
        
        for (let j = 0; j < a.length - 1; j++) {
            if (a[j] > a[j + 1]) {
                let temp = a[j]
                a[j] = a[j + 1]
                a[j + 1] = temp
                count += 1
            }
        }
        
    }

    console.log(`Array is sorted in ${count} swaps.`)
    console.log(`First Element: ${a[0]}`)
    console.log(`Last Element: ${a[a.length - 1]}`)
}
// 